01-04-2024/src/js/main.js

```diff
@@ -0,0 +1,64 @@
+import { contacts } from '../data/contacts.js';
```

Comment:
Very clean and cool implementation.
 **Challenge**
 Research debounce and why its important when using oninput patterns like the one you used.